data.js

```diff
@@ -1,57 +1,51 @@
-let Title;
-let Author;
-let bookList = [];
-
-const getValues = () => {
-  Title = document.querySelector('#title').value;
-  Author = document.querySelector('#author').value;
-  document.querySelector('#title').value = '';
-  document.querySelector('#author').value = '';
-};
-
-const addBook = () => {
-  document.getElementById('dynamic').innerHTML = '';
-  bookList.forEach((book, index) => {
-    document.getElementById('dynamic').innerHTML += `
-    <div class="books">
-      <p class="title">${book.title}</p>
-      <p class="authir">${book.author}</p>
-      <button class="remove" onClick = 'removeItem(${index})'>Remove</button>
-      <hr>
-    </div>`;
-  });
-};
-
-if (localStorage.getItem('Books')) {
-  bookList = JSON.parse(localStorage.getItem('Books'));
-  addBook();
+/* eslint-disable-next-line max-classes-per-file */
+class Book {
+  constructor(Title, Author) {
+    this.Title = Title;
+    this.Author = Author;
+  }
 }
 
-const storeBook = () => {
-  getValues();
-  const data = {
-    title: Title,
-    author: Author,
-  };
-  if (data.title !== '' && data.author !== '') {
-    bookList.push(data);
+class BookL {
+  bookList = [];
+
+  add() {
+    const Title = document.querySelector('#title').value;
+    const Author = document.querySelector('#author').value;
+
+    if (Title !== '' && Author !== '') {
+      const book = new Book(Title, Author);
+      this.bookList.push(book);
+      localStorage.setItem('Books', JSON.stringify(this.bookList));
+      this.render();
+    }
+    document.querySelector('#title').value = '';
+    document.querySelector('#author').value = '';
   }
-  Title = '';
-  Author = '';
-  localStorage.setItem('Books', JSON.stringify(bookList));
-  addBook();
-};
-
-const btnAdd = document.getElementById('btn');
 
-btnAdd.addEventListener('click', storeBook);
-
-const removeItem = (index) => {
-  bookList.splice(index, 1);
-  localStorage.setItem('Books', JSON.stringify(bookList));
-  addBook();
-};
+  render() {
+    document.getElementById('dynamic').innerHTML = '';
+    this.bookList = JSON.parse(localStorage.getItem('Books'));
+    this.bookList.forEach((book, index) => {
+      document.getElementById('dynamic').innerHTML += `
+      <div class="books">
+        <p class="title">${book.Title}</p>
+        <p class="authir">${book.Author}</p>
+        <button class="remove" onClick = 'bookl.remove(${index})'>Remove</button>
+        <hr>
+      </div>`;
+    });
+  }
 
-if (1 === 0) {
-  removeItem();
+  remove(index) {
+    this.bookList.splice(index, 1);
+    localStorage.setItem('Books', JSON.stringify(this.bookList));
+    this.render();
+  }
 }
+
+const bookl = new BookL();
+bookl.render();
+document.getElementById('btn').addEventListener('click', () => {
+  bookl.add();
+});
```

Comment:
- [ ] The`add` button is not working as intended. Neither does it update the local storage nor does it render a new book object on the DOM. Kindly fix this issue. Also, make sure the `remove` book feature is working as well as I cannot test it now since I cannot create any book object. :pray: